core/spec/models/spree/order/state_machine_spec.rb

```diff
@@ -180,9 +181,9 @@
 
       context "with shipped items" do
         before do
-          allow(order).to receive_messages :shipment_state => 'partial'
-          allow(order).to receive_messages :outstanding_balance? => false
-          allow(order).to receive_messages :payment_state => "paid"
+          allow(order).to receive_messages shipment_state: 'partial'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.